neon_core/skills/skill_manager.py

```diff
@@ -44,7 +44,7 @@ class NeonSkillManager(SkillManager):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        init_log(self.config, "neon-skills")
+        init_log(self.config, "skills")
```

Comment:
i dont think this should be called in init, it does not belong to an object, it should be in the entrypoint for `__main__` as it only should be done when the module is executed

Reply:
needs to happen somewhere in this thread though, otherwise log init isn't happening